client/infinite_scroll.js

```diff
@@ -9,11 +9,16 @@ TemplateController('infinite_scroll', {
       instance.target.trigger("becameVisible");
     }
     };
+    this.showMoreVisibleDebounced = () => {
+     return (event) => {
+       _.debounce(this.showMoreVisible, this.data.timeDelay);
+     }
+   };
   },
   onRendered() {
-    $(window).on('scroll',_.debounce(this.showMoreVisible, this.data.timeDelay));
+    $(window).on('scroll',this.showMoreVisibleDebounced());
```

Comment:
with the changes above, you just pass the function reference to jQuery:
`$(window).on('scroll',this.showMoreVisibleDebounced);`